src/App.js

```diff
@@ -30,10 +30,14 @@ class App extends Component {
   }
 
   render() {
+    if(this.state.error){
+      console.log(this.state.error)
+      return <h2 className='error-message'>Uh Oh, Something Went Wrong</h2>
```

Comment:
Ahh okay, so it renders the error instead of the entire 'main-page' element.  Good call!